src/test/java/com/lob/model/LetterTest.java

```diff
@@ -306,6 +307,41 @@ public void testCreateLetterWithMergeVariableObject() throws Exception {
         assertEquals("letter", letter.getObject());
     }
 
+    @Test(expected=InvalidRequestException.class)

```

Comment:
Should we just add the feature flag to the Java Lib account to test the success case of this?

Reply:
do you have an example in another sdk where we do it?

Reply:
No, looking through our libraries I do not see us do that. I was just wondering if we should test the success case in our ci, but it doesn't seem that common to add flagged features to our libraries in general. This seems fine as is.